packages/server/README.md

```diff
@@ -1,11 +1,144 @@
-# apollo-server-core
+# `@apollo/server`
 
-[![npm version](https://badge.fury.io/js/apollo-server-core.svg)](https://badge.fury.io/js/apollo-server-core)
-[![Build Status](https://circleci.com/gh/apollographql/apollo-server/tree/main.svg?style=svg)](https://circleci.com/gh/apollographql/apollo-server)
+## A TypeScript/JavaScript GraphQL server
 
-This package implements the core logic of Apollo Server. It exports a base version of `ApolloServer`. Typically you do not use this class directly but instead use an `ApolloServer` imported from the batteries-included `apollo-server` package or one of the integration packages like `apollo-server-express`.
+> Apollo Server 4 is currently in the alpha phase. Further changes (including backwards-incompatible changes) may be made before v4.0.0 is released. Docs for this version are available at https://www.apollographql.com/docs/apollo-server/v4/
 
-It also exports a set of plugins such as `ApolloServerPluginUsageReporting` which you can provide to the `plugins` option to the `ApolloServer` constructor.
+**Apollo Server is an [open-source](https://github.com/apollographql/apollo-server), spec-compliant GraphQL server** that's compatible with any GraphQL client, including [Apollo Client](https://www.apollographql.com/docs/react). It's the best way to build a production-ready, self-documenting GraphQL API that can use data from any source.
 
-[Read the docs.](https://www.apollographql.com/docs/apollo-server/)
-[Read the CHANGELOG.](https://github.com/apollographql/apollo-server/blob/main/CHANGELOG.md)
+
+You can use Apollo Server as:
+
+* A stand-alone GraphQL server
+* The GraphQL server for a [subgraph](https://www.apollographql.com/docs/federation/subgraphs) in a federated supergraph
+* The gateway for a [federated supergraph](https://www.apollographql.com/docs/federation/)
+
+Apollo Server provides a simple API for integrating with any Node.js web framework or serverless environment. The `@apollo/server` package itself ships with a standalone zero-dependency web server, and a middleware implementation for [the Express web framework](https://expressjs.com/). Integrations with other environments are community-maintained.
```

Comment:
The standalone package isn't zero dep though, and we might mention that it uses express internally?

Reply:
What about:
>ships with a standalone web server (built using Express)

If we get rid of the express-related deps we can remove that bit?